state-chain/pallets/cf-ingress-egress/src/lib.rs

```diff
@@ -1118,16 +1123,20 @@ pub mod pallet {
 					},
 				};
 
+			T::DepositHandler::on_deposit_made(deposit_details, deposit_amount);
+
+			// TODO: ensure minimum deposit?
+
+			// TODO: validate dca_params and refund_params
```

Comment:
do we have tickets for this?

Reply:
Im working on it in my PR, if i can get it in soon, then this can be completed in this PR.

Reply:
Created tickets now